server/bootstrap-template.yaml

```diff
@@ -218,6 +218,11 @@ Outputs:
     Value: !GetAtt Cognito.Outputs.CognitoOperationUsersUserPoolClientId
     Export:
       Name: "Serverless-SaaS-AdminUserPoolClientId" 
+  CognitoOperationUsersUserPoolId:
```

Comment:
I guess we can remove "Serverless-SaaS-AdminUserPoolProviderURL" output parameter